OpenBCI_32bit_Library.h

```diff
@@ -292,12 +316,24 @@ class OpenBCI_32bit_Library {
   // Variables
   boolean commandFromSPI;
   boolean firstDataPacket;
+  boolean isRunning;
+
+/* These variables are replaced by the new multichar commands
   boolean isProcessingIncomingSettingsChannel;
   boolean isProcessingIncomingSettingsLeadOff;
   boolean isProcessingIncomingSerialPassThru;
-  boolean isRunning;
   boolean settingBoardMode;
   boolean settingSampleRate;
+ */
+    
+// variables for the Multi char serial command management
```

Comment:
please remove comment and insert all alphabetically by type then variable name

Reply:
The rest all alphabetically by name (not by type) - will go with that strategy